.github/workflows/haskell.yml

```diff
@@ -61,11 +61,18 @@ jobs:
           flags: -external-libsodium-vrf
         EOF
 
+    - name: Select optimal cabal version
+      run: |
+        case "$OS" in
+          Windows_NT)   echo "CABAL_VERSION=3.2.0.0"      >> $GITHUB_ENV;;
+          *)            echo "CABAL_VERSION=3.4.0.0-rc4"  >> $GITHUB_ENV;;
```

Comment:
`>>` (append) instead of `>`?

Reply:
Yep.  As per: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions